packages/ethereum-storage/test/gas-price-definer.test.ts

```diff
@@ -37,7 +72,11 @@ describe('GasPriceDefiner', () => {
       expect(gasPrice).toBe(config.getDefaultEthereumGasPrice());
     });
 
-    it('returns the max of values returned by providers', async () => {
+
+   
+
+
```

Comment:
sorry and thanks  for notifying  that , finally i resolve  the issue of correct .prettierrrc for prettier-format command   in linting all the ts files , it will be applicable from the next commits